ComponentKit/Accessibility/CKComponentAccessibility.mm

```diff
@@ -24,7 +24,11 @@ static CKViewComponentAttributeValueMap ViewAttributesFromAccessibilityContext(c
     accessibilityAttributes[@selector(setAccessibilityIdentifier:)] = accessibilityContext.accessibilityIdentifier;
   }
   if (accessibilityContext.isAccessibilityElement) {
+#if TARGET_OS_IPHONE
     accessibilityAttributes[@selector(setIsAccessibilityElement:)] = accessibilityContext.isAccessibilityElement;
+#else
```

Comment:
I might prefer else-if's and then an #else with #error unrecognized platform... but that's probably overkill